trendyqc/trend_monitoring/models/bam_qc.py

```
@@ -406,3 +409,127 @@ class Duplication_metrics(models.Model):
     class Meta:
         app_label = "trend_monitoring"
         db_table = "duplication_metrics"
+
+
+class RNAseq_metrics(models.Model):
```

Ruff (0.8.0)

Model does not define `__str__` method

(DJ008)